tests/integrations/data/local-area-shielding-advice.csv

```diff
@@ -0,0 +1,12 @@
+ladcode,ladname,shielding_advice,start_datetime_utc
+E06000002,Middlesbrough Borough Council,YES,2020-12-02T00:00:01
+E06000002,Middlesbrough Borough Council,NO,2020-12-10T00:00:01
+E06000003,Dummy Council,NO,2020-12-02T00:00:01
+E06000003,Dummy Council,YES,2020-12-10T00:00:01
+E08000012,Liverpool City Council,YES,2020-08-20T00:00:01
+E08000015,Wirral Borough Council,YES,2020-12-02T00:00:01
+E08000015,Wirral Borough Council,NO,2021-12-02T00:00:01
+E06000018,Nottingham City Council,NO,2020-12-02T00:00:01
+E06000018,Nottingham City Council,YES,2020-12-10T00:00:01
+E0800021,Dummy Council,NO,2020-12-02T00:00:01
+E0800021,Dummy Council,YES,2022-12-02T00:00:01
```

Comment:
I've been using `freezegun` to test these data in the future type things. Although I guess we might hope that if this service is still around at the end of 2022 that there are bigger things to worry about.

Reply:
tbh, i am hoping that this whole integration should be moved to backend by mid/end of January... otherwise its just additional work we will need doing keeping data consistent for front end and (inevitably) MI.